prow/config/jobs/istio.yaml

```diff
@@ -208,6 +208,11 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: " --istio.test.retries=1 "
 
+  - name: install-cni-test
+    modifiers: [hidden, optional]
```

Comment:
What does `hidden` mean? I'd like this job results to appear on github UI so that we are aware of its results.
+1 for keeping it optional for a while, once it proves stabilized we can make it required.